### DISCUSSION TITLE
Custom goesproc configs

### DISCUSSION THREAD

**Discussion by JVital2013** [JVital2013/goestools-win#7] [Show and tell]
Do you have a custom goesproc config that you like to use? Share it here!

Please zip up the config so it can be used to replace the default config in Goestools for Windows

**Comment by JVital2013** [comment]
**Store all saved data in dated subfolders**, instead of one big folder for each product. Matches the default functionality of Goestools for Linux

[goesproc-goesr.zip](https://github.com/JVital2013/goestools-win/files/12209802/goesproc-goesr.zip)

**Comment by juanjesus2023** [comment]
Thank you very much for the work done. I've been working with goestools on debian for a few months and I'm very excited about the idea of having it on windows. Detect that in the 32bit version the borders do appear and not in the 64bit version.

**Reply by JVital2013** [reply]
Hello, thanks for giving the Windows version a try!

Borders do work on both the 32bit and 64bit version, but I'm sorry to hear you're having issues with borders on the 64-bit version. I have two thoughts on things to check:

1. **Are you running the latest version of Goestools for Windows?** There was a similar bug in older versions, but version 1.0.2 and newer work fine.
2. **Is the 64-bit Microsoft Visual C++ Redistributable Up-To-Date?** The 32-bit and 64-bit installs are separate, and ideally they should be version 14.36.32532

![image](https://github.com/JVital2013/goestools-win/assets/24253715/4b443f40-fe12-4844-bcde-82bd9548048a)

**Comment by tmabell** [comment]
Is there a way that we can save data to more than one location?  Presently goesproc-goesr.conf will only allow one file path for text files.  Adding a second path causes the software to error out.

**Reply by JVital2013** [reply]
To do this, create a copy of the entire handler, and specify a new patch within the copy. This example will save text files to two folders:

```ini
# Miscellaneous text.
[[handler]]
type = "text"
origin = "other"
directory = "Saved/text"
filename = "{time:%Y%m%dT%H%M%SZ}_{filename}"
json = false

# Miscellaneous text copy
[[handler]]
type = "text"
origin = "other"
directory = "Saved/anothercopy"
filename = "{time:%Y%m%dT%H%M%SZ}_{filename}"
json = false
```

**Comment by tmabell** [comment]
I’ve run into a snag.  This is my first attempt but I am getting a “permission denied” error for the second emwin path.  I have confirmed that the correct permissions are set on drive D.  Once the error occurs, the program crashes with this “Press any key to continue”  I apologize for being less than savvy with Linux! 

 �

# EMWIN

[[handler]]

type = "emwin"

directory = "/Program Files/WxMesgNet/WxData/RxFiles"

 �

# EMWIN Copy

[[handler]]

type = "emwin"

directory = "D:/WxData/Emwin"

 �

 �

 �

**Reply by JVital2013** [reply]
Looks like you found a bug! I get the same error. I'll do some debugging and will get back to you.

**Reply by JVital2013** [reply]
@tmabell please update to v1.0.7 to fix this issue. `D:/WxData/Emwin` should not cause a crash anymore

**Reply by tmabell** [reply]
Updated, tested and confirmed fixed!  Thank you @JVital2013

**Comment by tmabell** [comment]
Has anyone been able to nail down the cause of frequent dropped packets?  I'm not sure how prevalent this problem is but it seems to occur on random systems.
